external_contact.go

```diff
@@ -29,3 +29,55 @@ func (c *WorkwxApp) RemarkExternalContact(req *ExternalContactRemark) error {
 	})
 	return err
 }
+
+// GetExternalContactCorpTagList 获取企业标签库
+func (c *WorkwxApp) GetExternalContactCorpTagList(tagID ...string) ([]ExternalContactCorpTagGroup, error) {
```

Comment:
`ListExternalContactCorpTags` for idiomatic English naming?

Reply:
at fisrt i named `ListExternalContactCorpTags` but reference url `get_corp_tag_list` i choose `GetExternalContactCorpTagList`,  luckly we see eye to eye